lib/person.js

```diff
@@ -0,0 +1,10 @@
+var EventEmitter = require('events').EventEmitter;
```

Comment:
you have used eventEmitter. For sending messages. Its good. But you should also try it out with socket.io to get the socket working understanding.